src/main/java/nl/knaw/dans/migration/core/tables/ExpectedDataset.java

```diff
@@ -130,17 +146,28 @@ public void setLicenseName(@Nullable String licenseName) {
         this.licenseName = licenseName;
     }
 
+    @Nullable
+    public int getExpectedVersions() {
+        return expectedVersions;
+    }
+
+    public void setExpectedVersions(@Nullable int expectedVersions) {
+        this.expectedVersions = expectedVersions;
+    }
+
     @Override
     public String toString() {
         return "ExpectedDataset{" +
             "doi='" + doi + '\'' +
+            ", expectedFilesDoi='" + expectedFilesDoi + '\'' +
```

Comment:
what is the meaning of this `expectedFilesDoi`? For fedora datasets this is always the same as the `doi`. And for vault datasets, the `convert-bag-to-deposit` makes sure we use the first-doi as the doi for both versions.

Reply:
I forgot the latter fact. With same DOI through the whole bag-sequence this indeed has no purpose.